src/site/content/en/metrics/inp/index.md

```diff
@@ -44,20 +44,88 @@ This article explains how INP works, how to measure it, and offers advice for im
-{% endDetails %}
+<div class="table-wrapper scrollbar">
+  <table>
+    <thead>
+      <tr>
+        <th>Number of page interactions</th>
+        <th>INP candidate determined as a page's INP</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1-49</td>
+        <td>Largest</td>
+      </tr>
+      <tr>
+        <td>50-99</td>
+        <td>2nd Largest</td>
+      </tr>
+      <tr>
+        <td>100-149</td>
+        <td>3rd Largest</td>
+      </tr>
+      <tr>
+        <td>150-199</td>
+        <td>4th Largest</td>
+      </tr>
+      <tr>
+        <td>200-249</td>
+        <td>5th Largest</td>
+      </tr>
+      <tr>
+        <td>250-299</td>
+        <td>6th Largest</td>
+      </tr>
+      <tr>
+        <td>300-349</td>
+        <td>7th Largest</td>
+      </tr>
+      <tr>
+        <td>350-399</td>
+        <td>8th Largest</td>
+      </tr>
+      <tr>
+        <td>400-449</td>
+        <td>9th Largest</td>
+      </tr>
+      <tr>
+        <td>450+</td>
+        <td>10th Largest</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
 
-An _interaction_ is a group of event handlers that fire during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`. An interaction can be driven by JavaScript, CSS, built-in browser controls (such as form elements), or a combination thereof.
+{% endDetails %}
 
-An interaction's latency consists of the single longest [duration](https://w3c.github.io/event-timing/#ref-for-dom-performanceentry-duration%E2%91%A1:~:text=The%20Event%20Timing%20API%20exposes%20a%20duration%20value%2C%20which%20is%20meant%20to%20be%20the%20time%20from%20when%20user%20interaction%20occurs%20(estimated%20via%20the%20Event%27s%20timeStamp)%20to%20the%20next%20time%20the%20rendering%20of%20the%20Event%27s%20relevant%20global%20object%27s%20associated%20Document%E2%80%99s%20is%20updated) of a group of event handlers that drives the interaction, from the time the user begins the interaction to the moment the next frame is presented with visual feedback.
+An interaction's latency consists of the single longest [duration](https://w3c.github.io/event-timing/#ref-for-dom-performanceentry-duration%E2%91%A1:~:text=The%20Event%20Timing%20API%20exposes%20a%20duration%20value%2C%20which%20is%20meant%20to%20be%20the%20time%20from%20when%20user%20interaction%20occurs%20(estimated%20via%20the%20Event%27s%20timeStamp)%20to%20the%20next%20time%20the%20rendering%20of%20the%20Event%27s%20relevant%20global%20object%27s%20associated%20Document%E2%80%99s%20is%20updated) of a group of event callbacks that drives the interaction, from the time the user begins the interaction to the moment the next frame is presented with visual feedback.
```

Comment:
I find these long fragment search URLs quite brittle btw. They break as soon as any text is tweaked on the page. Should it just link to the duration link that page links to: https://w3c.github.io/performance-timeline/#dom-performanceentry-duration